public/server/db.json

```diff
@@ -72,6 +72,10 @@
   "bff": {
     "definition": "Best Friend Forever"
   },
+  "bf": {
```

Comment:
You didn't need to additionally add this to `db.json`. We no longer use this file to store abbreviations.